msal/src/androidTest/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name">com.microsoft.identity.msal.test</string>
```

Comment:
Sorry i'm not following, are you saying the tests run are failing or? We didn't see any failures before in the dev branch.